### ISSUE TITLE
Accessibility issue: The web page should have the content's human language indicated in the markup (1)

### ISSUE THREAD

**Issue by ameliejyc** [mineshmshah/mineshmshah.github.io#4]
Solve this by adding lang tag in the html e.g. EN for English.
